src/screens/Setup/BirthDate.js

```diff
@@ -22,37 +23,71 @@ export default class BirthDate extends Component {
     const birthDate = navigation.getParam('birthDate')
     const country = navigation.getParam('country')
     const state = navigation.getParam('state')
+    const countryName = navigation.getParam('countryName')
+    const stateName = navigation.getParam('stateName')
 
     this.state = {
-      birthDate,
-      country,
-      state,
+      birthDate: birthDate,
```

Comment:
you don't need to use key: value pattern when the variable has the same name as the key, you can change this to:
```suggestion
      birthDate,
      country,
      state,
      countryName,
      stateName,
```

Reply:
same for other cases below

Reply:
Done!